### ISSUE TITLE
False positive for strings

### ISSUE THREAD

**Issue by heckad** [snok/flake8-type-checking#55]
```python
from typing import TYPE_CHECKING
from weakref import WeakKeyDictionary

if TYPE_CHECKING:
    from typing import Any # TC004 Move import 'Any' out of type-checking block. Import is used for more than type hinting.


d = WeakKeyDictionary["Any", "Any"]()
```

**Comment by sondrelg** [comment]
Thanks for reporting this @heckad. You are right, there was a mistake in the way we recorded what imports were used for.

I just released a new version with a fix (v1.1.2). Would you mind checking if the new version works for you?

**Comment by heckad** [comment]
Yes, now it's work. Thank you very much for the quick fix!``
